testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/messaging/jms/context/notClosingInjectedContext/auxiliary/Ejb.java

```diff
@@ -55,7 +57,12 @@ public void send(String text) {
     private void send(String text, JMSContext jmsContext) {
         try {
             LOGGER.info("Sending: " + text);
-            jmsContext.createProducer().send(lookup(Mdb.JNDI_NAME), text);
+            long start = System.currentTimeMillis();
+            Queue queue = lookup(Mdb.JNDI_NAME);
+            while(queue == null && (System.currentTimeMillis() - start < TIMEOUT)) {
+                queue = lookup(Mdb.JNDI_NAME);
```

Comment:
@ehsavoie I'd add a little sleep after the lookup, say 200ms

Reply:
I agree :)